homeassistant/components/calendar/__init__.py

```diff
@@ -69,11 +69,18 @@ def normalize_event(event):
     normalized_event["end"] = end
 
     # cleanup the string so we don't have a bunch of double+ spaces
+    normalized_event["event_id"] = event.get("id", "")
+    normalized_event["status"] = event.get("status", "")
+    normalized_event["created"] = event.get("created", "")
+    normalized_event["updated"] = event.get("updated", "")
```

Comment:
This is not normalized, this is more a placeholder because it's not a valid datetime.